### DISCUSSION TITLE
增加按照字符数收费

### DISCUSSION THREAD

**Discussion by hanxuebing** [immersive-translate/immersive-translate#2345] [Ideas]
### 新功能 | New Feature

1.收费能否增加按照字符数的选项

### 为什么您想要添加这些功能 | Why you want to add these features

1.使用率较低用户按月订阅不划算

### 提议 | Proposal

需要收集用户翻译的字数可能涉及一些不可预料问题，增加工作量

**Comment by theowenyoung** [comment]
您可直接从第三方购买 api key 即可按字数收费～
